src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs

```diff
@@ -462,7 +466,11 @@ public static CultureInfo InvariantCulture
         {
             get
             {
+#if TARGET_BROWSER
```

Comment:
I would suggest adding a link to the issue also. And in the PR description add the reasoning for why this is needed, based on the discussion in that issue.